### PR TITLE
Post/new devcenter

### PR DESCRIPTION
**Pull request by viktorbenei**

### CODE REVIEW

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+and distribute it with `bitrise run deploy-to-github-pages`.
+
+The current "compiled" static site can also be found on the
+[gh-pages branch](https://github.com/bitrise-io/devcenter/tree/gh-pages)
+of the GitHub repository.
+
+We use a wide variety of tools for our services,
+for example our Blog (which is also [open source](https://github.com/bitrise-io/blog))
+uses [middleman](https://middlemanapp.com/), also compiled to a static site
+which is hosted on Amazon S3. You can find a related guide guide,
```
Comment (birmacher) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: guide guide -> guide

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+well, [on our DevCenter](http://devcenter.bitrise.io/tutorials/auto-deploy-middleman-project/) ;)
+
+For the DevCenter we wanted to try another tool, so we went with [MkDocs](http://www.mkdocs.org/)
+for organizing and building the code, and [GitHub Pages](https://pages.github.com/)
+for hosting the static site. This was our first project to utilize GitHub Pages,
+and we were plesently surprised. Deploying the repository to GitHub Pages
+is quite straight forward, once you understand the two types of sources it supports:
```
Comment (birmacher) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: straightforward

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+For example, the [Bitrise CLI and bitrise.yml](http://devcenter.bitrise.io/bitrise-cli/)
+section was rewritten from the ground up. When we wrote the original section
+for our previous DevCenter we were still kind of new to the CLI, as the
+current [open source Bitrise CLI](https://www.bitrise.io/cli) just replaced
+our previous, closed source solution. A lot has changed since, the CLI
+evolved rapidly, especially since we switched to
+[monthly releases](http://blog.bitrise.io/2016/04/13/new-version-of-the-bitrise-cli-1-3-0-is-finally-released.html),
+and we now automate the development, testing and distribution of the [bitrise.io](https://www.bitrise.io)
+website, our [blog](https://github.com/bitrise-io/blog) and now the new [DevCenter](https://github.com/bitrise-io/devcenter),
+in addition to lots of mobile app and [sample projects](https://github.com/bitrise-samples)
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: mobile apps

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+using the `Bitrise CLI`.
+
+With the updated DevCenter we tried our best to guide you through the vast `Bitrise` ecosystem,
+including both the [bitrise.io](https://www.bitrise.io) web service and our
+[open source build runner / CLI](https://www.bitrise.io/cli), and
+__we're determined to keep it that way__.
+So, __if you'd have any question or issue with any page of the DevCenter__,
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: questions or issues with any page on the DevCenter

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+We use a wide variety of tools for our services,
+for example our Blog (which is also [open source](https://github.com/bitrise-io/blog))
+uses [middleman](https://middlemanapp.com/), also compiled to a static site
+which is hosted on Amazon S3. You can find a related guide guide,
+well, [on our DevCenter](http://devcenter.bitrise.io/tutorials/auto-deploy-middleman-project/) ;)
+
+For the DevCenter we wanted to try another tool, so we went with [MkDocs](http://www.mkdocs.org/)
+for organizing and building the code, and [GitHub Pages](https://pages.github.com/)
+for hosting the static site. This was our first project to utilize GitHub Pages,
+and we were plesently surprised. Deploying the repository to GitHub Pages
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: pleasantly

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+is quite straight forward, once you understand the two types of sources it supports:
+
+1. You can use GitHub Pages' built in [Jekyll](https://jekyllrb.com/) feature.
+1. Or you can build your static HTML, css, js, .. resources, and store that on the `gh-pages` branch directly.
+
+As we decided to use `MkDocs` and we wanted to keep our options open (if we'd
+decide to host the devcenter somewhere else in the future) we went with the second option.
+Generating all the static resources with `MkDocs` is as simpel as running `mkdocs build --clean`,
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: simple

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+which by default will generate the files into a `site/` subdirectory.
+All we had to automate is to run `mkdocs build --clean`, and sync the `site/` directory
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: running

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+We were thrilled, as we could figure out these details in less than an hour,
+and everything seemed to work properly! __But then we went ahead and published a change__
+and a new release of the DevCenter with some minor
+modifications, went to check it on [devcenter.bitrise.io](http://devcenter.bitrise.io/)
+and... __The site was gone O_O__
+
+To be precise, when we visited [devcenter.bitrise.io](http://devcenter.bitrise.io/)
+the 404 page of `bitrise-io.github.io` was presented. We were surprised
+and through we messed something up. Double checked the configurations, and
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: thought

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown (hunk 6 above)
Comment (viktorbenei) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: I like my version better :P

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown (hunk 5 above)
Comment (viktorbenei) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: ahh, I wanted to write peasant (JK - I'll fix this)

File: source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown [hunk 9]
```diff
@@ -0,0 +1,144 @@
+---
+title: The New DevCenter finally arrived!!
```
Comment (bloodrabbit) [inline source/2016-12-12-the-new-devcenter-finally-arrived.html.markdown]: has arrived

**Pull request merged**